app/views/contribute.html

```diff
@@ -0,0 +1,186 @@
+        help you learn and develop your skills with the kit
+      </li>
+      <li>
+        we will credit you in our update notes and other supporting information
+        unless you tell us not to (for example, because you want to stay
+        anonymous)
+      </li>
+    </ul>
+    <p>
+      If you are employed by NHS England, you will usually be able to use
+      examples of contributing to the kit as part of your yearly performance
+      review - talk to your line manager about how to include it in your
+      performance objectives.
+    </p>
+    <h2 id="how-you-can-contribute-to-the-kit">
+      How you can contribute to the kit
+    </h2>
+    <p>
+      There are many different ways that you can contribute, depending on your
+      skill and interests. You do not have to be an expert in coding.
+    </p>
+    <p>You can:</p>
+    <ul>
+      <li>add or improve content for people using the kit</li>
+      <li>
+        do quality assurance or accessibility testing of new features or
+        guidance
+      </li>
+      <li>fix bugs</li>
+      <li>add new features</li>
+      <li>improve the contribution process</li>
+    </ul>
+
+    <h3 id="add-or-improve-documentation-for-people-using-the-kit">
+      Add or improve content for people using the kit
+    </h3>
+    <p>You can also help improve the content for:</p>
+    <ul>
+      <li>this document site</li>
+      <li>the pages that people view when they download and install the kit</li>
+    </ul>
+    <p>
+      This includes: writing to the
+      <a href="https://service-manual.nhs.uk/content">NHS content guide</a> (and
+      <a
+        href="https://github.com/nhsuk/nhsuk.service-manual.prototype-kit.docs/blob/main/README.md"
+        >standardising terms that we only use in the kit</a
+      >) and making the structure of the site easy to use.
+    </p>
+    <p>
+      Existing things we want to do for guidance are labelled &#39;<a
+        href="{{board}}{{label}}Content"
+        >content</a
+      >&#39; on our project board on on the lists of things we want to do for
+      the kit. You can also add your own.
+    </p>
+    <h3 id="do-quality-assurance-or-accessibility-testing">
+      Do quality assurance or accessibility testing
+    </h3>
+    <p>
+      You can make sure that the code in our kit is robust and accessible by:
+    </p>
+    <ul>
+      <li>
+        running automated and manual tests against WCAG 2.2 AA specifications
+      </li>
+      <li>helping us set up pipelines for automated testing in GitHub</li>
+    </ul>
+    <p>
+      Accessibility issues are labelled &#39;<a
+        href="{{board}}{{label}}Accessibility"
+        >accessibility</a
+      >&#39; on our project board on on the lists of things we want to do for
+      the kit. You can also add your own.
+    </p>
+    <h3
+      id="fix-bugs-or-build-new-features-enhancements-for-the-nhs-prototype-kit"
+    >
+      Fix bugs or build new features (&#39;enhancements&#39;) for the NHS
```

Comment:
Not sure we need to use the term 'enhancements'?

```suggestion
      Fix bugs or build new features for the NHS
```

Reply:
as per below - moving away from 'enhancements' starts to deviate from the contribution system used by the rest of the NHS service manual team - which is why I'd rather leave that for Sara to make a call on in case it means wholesale changes across the entire contribution system